tasks/cdn-cli.js

```diff
@@ -0,0 +1,188 @@
+import chalk from 'chalk';
+import textTable from 'text-table';
+import { toCdnEntryName } from './cdn-entry-name.js';
+import { CDN_ENVIRONMENTS } from './cdn-environments.js';
+import { CdnManager } from './cdn-manager.js';
+import { NO_CACHE } from './cellar-client.js';
+import { getCurrentBranch } from './git-utils.js';
+
+async function run () {
+  const args = getArgs();
```

Comment:
suggestion: maybe it's just me but I would inline `getArgs` and destructure them directly like this:

```js
const [command, env, name] = process.argv.slice(2);
```

`getCdnEnvironment` only needs the `env` and `getCdnEntryName` only needs the `name`.